### PR TITLE
Update Get-SPOUserOneDriveLocation.md

### PR DESCRIPTION
**Pull request by PowershellScripts**
Suggested text for missing parameter description.

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOUserOneDriveLocation.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Get the current location the user's OneDrive location, url, and site ID.
 
 ### -UserPrincipalName
 
-PARAMVALUE: String
+UserPrincipalName or UPN defined for the specific user on the SPO tenant
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOUserOneDriveLocation.md:43]:
```suggestion
UserPrincipalName or UPN defined for the specific user on the SPO tenant.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PowershellScripts, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@PowershellScripts Gentle reminder, please implement the suggested changes. thanks

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**